### PR TITLE
Create manual-indexes.adoc

### PR DESCRIPTION
**Pull request by rohankharwar**
Kindly review and let me know your comments.

### CODE REVIEW

File: docs/manual-indexes.adoc [hunk 1]
```diff
@@ -0,0 +1,166 @@
+= Manual Indexes
+
+== Data Used
+
+In the below examples I used the flight data setup from the following link:
+
+https://github.com/nicolewhite/neo4j-flights
+
+
+== Enable Manual Indexes on Node and Relationship Property
```
Comment (jexp) [inline docs/manual-indexes.adoc]: Actually those are about auto-indexes, which are a kind of legacy-index but not the ones I was focusing on in apoc. Apoc has it's own procedures for adding things to the indexes which are meant to be called in batch post-creation or better as par of the create statements.

I'd rather prefer not to have legacy-autoindexes.

File: docs/manual-indexes.adoc [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+Once the auto indexes are enabled.
+
+One will have to add the relationship property to the index.
+This can be achieved using the APOC stored procedure - `apoc.index.addRelationship`.
+
+----
+match (f:Flight)-[r:DESTINATION]->(a:Airport)
+with r
+call apoc.index.addRelationship(r,["taxi_time"])
```
Comment (jexp) [inline docs/manual-indexes.adoc]: also this works differently than the auto-index b/c the index-name is the relationship-type and here you only add one property whereas in the auto-index you had two rel-properties indexed

File: docs/manual-indexes.adoc [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+
+Once the relationship index is created we can start using it.
+Here is one example of using this without Stored Procedure.
+
+
+This query will give me the count of the relationship `DESTINATION` with the property `taxi_time = 11`.
+
+----
+START r=relationship:DESTINATION(taxi_time="11")
```
Comment (jexp) [inline docs/manual-indexes.adoc]: I wouldn't want to use start anymore. It's meant to go away. And the index procedures do the same but better.

File: docs/manual-indexes.adoc [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+return node
+----
+
+To Remove the manual index `DESTINATION` created -
+
+----
+CALL apoc.index.remove('DESTINATION') YIELD type,name,config
+----
+
+== Adding Manual Index on Node Property
```
Comment (jexp) [inline docs/manual-indexes.adoc]: I would start with nodes and then discuss relationships next. Node indexes are easier to understand.

File: docs/manual-indexes.adoc [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+Once the auto indexes are enabled.
+
+One will have to load the nodes to the index.
+This can be achieved using one of the APOC stored procedure - `apoc.index.addNode`.
+
+----
+match (f:Flight)
```
Comment (jexp) [inline docs/manual-indexes.adoc]: perhaps promote to run the apoc.index.addNode as part of the create or load csv script.

File: docs/manual-indexes.adoc [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+call apoc.index.addNode(f,["flight_num"])
+return count(f)
+----
+
+The above statement will create the auto index with the same name as the Label name in this case `Flight` and add the relationships to the index.
```
Comment (jexp) [inline docs/manual-indexes.adoc]: add the _nodes_

File: docs/manual-indexes.adoc [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+----
+CALL apoc.index.list() YIELD type,name,config
+----
+
+Output:
+
+image::https://github.com/neo4j-contrib/neo4j-apoc-procedures/tree/master/docs/img/apoc-manual-indexes-nodes.png[width=500]
+
+
+Once the relationship index is created we can start using it.
```
Comment (jexp) [inline docs/manual-indexes.adoc]: _node_ index

File: docs/manual-indexes.adoc [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+----
+call apoc.index.nodes('Flight','flight_num:43') YIELD node
+match (node)-[:DESTINATION]->(a:Airport)-[:IN_CITY]->(c:City)
+return distinct a.name, c.name limit 10
+----
+
+To Remove the manual index `Flight` created -
+
+----
+CALL apoc.index.remove('Flight') YIELD type,name,config
```
Comment (jexp) [inline docs/manual-indexes.adoc]: you don't need yield for a *plain* call

File: docs/manual-indexes.adoc [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+----
+match (a:Flight {flight_num:43}) with a
+call apoc.index.out(a,"DESTINATION","taxi_time:11") YIELD node
+return node
+----
+
+To Remove the manual index `DESTINATION` created -
+
+----
+CALL apoc.index.remove('DESTINATION') YIELD type,name,config
```
Comment (jexp) [inline docs/manual-indexes.adoc]: you don't need yield here.

File: docs/manual-indexes.adoc [hunk 10]
```diff
@@ -0,0 +1,166 @@
+= Manual Indexes
+
+== Data Used
+
+In the below examples I used the flight data setup from the following link:
+
+https://github.com/nicolewhite/neo4j-flights
```
Comment (jexp) [inline docs/manual-indexes.adoc]: perhaps we can include a subset of the load script as we plan to turn this manual into individual browser guides, then they should be self-contained.

**Comment by rohankharwar** [comment]
Hi @jexp: Updated the images and the implemented the changes to the doc.

**Comment by jexp** [comment]
Merged manually, thanks a lot @rohankharwar

**Pull request closed without merging**